### ISSUE TITLE
VTKLattice: support wrapping StructuredPoints

### ISSUE THREAD

**Issue by itziakos** [simphony/simphony-mayavi#73]

**Comment by stefanoborini** [comment]
I don't think this requires any change. StructuredPoints is a derived class of ImageData, with only additional constraints for the extent and the origin. Considering that the check of data_set is over isinstance(data_set, ImageData), this is satisfied by StructuredPoints as well. 

I will add a test to verify this is true, but is it useful as a functionality?

**Comment by itziakos** [comment]
If I remember correctly not all lattices can be represented as StructuredPoints

**Comment by stefanoborini** [comment]
For those, we use PolyData in fact.
